build-system/tasks/e2e/selenium-webdriver-controller.js

```diff
@@ -242,7 +242,7 @@ class SeleniumWebDriverController {
    */
   async getActiveElement() {
     const root = await this.getRoot_();
-    const getter = root => root.parentNode.activeElement;
+    const getter = root => root.ownerDocument.activeElement;
```

Comment:
I think we discovered that `ownerDocument.activeElement` will be the host element and not the focused element inside the ShadowDOM, is that what you want? To get the focused element itself you would need to use `root.activeElement || root.ownerDocument.activeElement`

Reply:
this was sloppy copy paste, Done.